median/plugin.go

```diff
@@ -40,6 +41,15 @@ func (p *Plugin) NewMedianFactory(ctx context.Context, provider types.MedianProv
 
 	includeGasPriceSubunitsInObservation := !isZeroDataSource
 
+	var deviationFunc median.DeviationFunc
+	if deviationFuncDefinition != nil {
```

Comment:
Could make this tolerant to empty maps? So it doesn't surprise anyone
```suggestion
	if len(deviationFuncDefinition) == 0 {
```